services/wallet/commands.go

```diff
@@ -648,6 +648,10 @@ func findFirstRange(c context.Context, account common.Address, initialTo *big.In
 	to := initialTo
 	goal := uint64(20)
 
+	if from.Cmp(to) == 0 {
+		return to, nil
+	}
+
 	firstNonce, err := client.NonceAt(c, account, to)
```

Comment:
without new `if` block this request was executed although there is no point in in checking `0 - 0` range